packages/trader/src/AppV2/Components/OnboardingGuide/GuideForPages/onboarding-guide.tsx

```diff
@@ -15,12 +15,18 @@ const OnboardingGuide = ({ type = 'trade_page' }: TOnboardingGuideProps) => {
     const guide_timeout_ref = React.useRef<ReturnType<typeof setTimeout>>();
     const is_button_clicked_ref = React.useRef(false);
 
-    const [guide_dtrader_v2, setGuideDtraderV2] = useLocalStorageData<boolean>(`guide_dtrader_v2_${type}`, false);
+    const [guide_dtrader_v2, setGuideDtraderV2] = useLocalStorageData<Record<string, boolean>>('guide_dtrader_v2', {
+        trade_types_selection: false,
```

Comment:
Instead of storing 3 flags in localStorage now we'll have object:
```
'guide_dtrader_v2', {
        trade_types_selection: false,
        trade_page: false,
        positions_page: false,
    }
```